packages/spotlight/Spottable/Spottable.js

```diff
@@ -408,8 +402,10 @@ const Spottable = hoc(defaultConfig, (config, Wrapped) => {
 		}
 
 		render () {
-			const {disabled, spotlightId, ...rest} = this.props;
-			const spottable = this.state.focusedWhenDisabled || isSpottable(this.props);
+			const {disabled, spotlightId, spotlightDisabled, ...rest} = this.props;
+			this.focusedWhenDisabled = this.isFocused && (disabled || spotlightDisabled);
+			const spottable = this.focusedWhenDisabled || isSpottable(this.props);
```

Comment:
I'm concerned that rendering based off an instance property value `this.isFocused`/`this.focusedWhenDisabled` will cause inconsistent renders, since it's not guaranteed to re-render when these values change.

Part of my concern is with the way we're attempting to manage the two different `focusedWhenDisabled` states (`this.focusedWhenDisabled` and `this.state.focusedWhenDisabled`). It looks like it would be easy for these to become out of sync (see my comment above) or not be able to provide the current value needed within the given method or life-cycle. It seems like we could utilize the existing state property and adjust the behavior to allow or prevent updates (resulting in unwanted renders) in `sCU`. Thoughts?

Reply:
Yeah I see the concern with having two separate variables.  What would you think about instead of using state, we could technically do a `this.forceUpdate` and not have any state? The reason for not using state, is because we need to mutate in the render method so `setState` would be bad there, but this way we could avoid `sCU` all together.

Reply:
With `this.forceUpdate` we can address the above comment while still avoiding lifecycle of `setState`